src/Widgets/BreadcrumbWidget/BreadcrumbWidgetEditingConfig.ts

```diff
@@ -0,0 +1,6 @@
+import { provideEditingConfig } from 'scrivito'
+import { BreadcrumbWidget } from './BreadcrumbWidgetClass'
+
+provideEditingConfig(BreadcrumbWidget, {
+  title: 'Breadcrumb',
```

Comment:
* [ ] FYI terminology: it's either "Breadcrumbs" or "Breadcrumb Navigation". In case we'd like to improve, please keep class names/filenames in sync.

Reply:
The singular "Breadcrumb" is also in use, e.g. https://getbootstrap.com/docs/5.3/components/breadcrumb/ or https://developer.mozilla.org/en-US/docs/Glossary/Breadcrumb. I'll leave it as is.